src/services/FulfillmentService.js

```diff
@@ -29,12 +30,18 @@ class FulfillmentService {
     try {
       const {
         params: { slug },
-        body: { type, value },
+        body: { location },
       } = req;
-
-      // TODO: get items by slug,type and value
-
-      return organizations;
+      const type = slug.split('/')[1] || '';
+      const county = location || '';
+      const url = `${process.env.DRUPAL_URL}/rest/v1/fulfillments/services/${type}/${county}`;
```

Comment:
I don't see this URL in the Drupal Views. How is this going to work? Also, how is the slug going to map to a service in Drupal?